### ISSUE TITLE
Level Pool using both inflow and previous timestep inflow

### ISSUE THREAD

**Issue by jdmattern-noaa** [NCAR/wrf_hydro_nwm_public#326]
The Level Pool physics code uses inflow from the current timestep and the previous timestep. It is unclear if the actual previous timestep inflow is passed to Level Pool.

My unit tests match a simulation output only when I pass the current inflow to both the current timestep inflow and the previous timestep inflow in Level Pool. Here is the call to Level Pool in Channel Routing:

https://github.com/NCAR/wrf_hydro_nwm_public/blob/master/trunk/NDHMS/Routing/module_channel_routing.F#L1899

Here is a comment in Channel Routing that seems to consider the current timestep inflow passed to Level Pool as the actual previous timestep inflow:

https://github.com/NCAR/wrf_hydro_nwm_public/blob/master/trunk/NDHMS/Routing/module_channel_routing.F#L1869

If this is the case, would the previous timestep inflow passed to Level Pools actually be from two timesteps back?

**Comment by kafitzgerald** [comment]
@jdmattern-noaa was this fixed in one of your recent PRs?

**Comment by jdmattern-noaa** [comment]
No. This is more of a question on our part for clarification. Nothing in my code would need to be updated based upon this.
